src/uct/api/uct.h

```diff
@@ -1817,13 +1819,14 @@ UCT_INLINE_API ssize_t uct_ep_tag_eager_bcopy(uct_ep_h ep, uct_tag_t tag,
 
 /**
  * @ingroup UCT_TAG
- * @brief Send tagged message using zcopy eager protocol.
+ * @brief Zcopy eager tagged-send operation.
  *
- * Eager protocol means that the whole data is sent immediately to the peer.
- * The input data (which has to be previously registered) in @a iov array of
- * @ref uct_iov_t structures sent to remote side ("gather output"). Buffers
- * in @a iov are processed in array order. This means that the function complete
- * iov[0] before proceeding to iov[1], and so on.
+ * This routine sends a message using zero copy eager protocol. Eager protocol
+ * means that the whole data is sent immediately to the peer. The input data
+ * (which has to be previously registered) in @a iov array of @ref uct_iov_t
+ * structures sent to remote side ("gather output"). Buffers in @a iov are
+ * processed in array order, so the function complete iov[0] before proceeding
+ * to iov[1], and so on.
```

Comment:
Again. Please elaborate what is the difference between this one , and two other eager protocols.